README.md

```diff
@@ -1,5 +1,116 @@
-# Re:Earth
 
-- [Discussions](https://github.com/reearth/reearth/discussions)
-- Frontend (web): [reearth/reearth-web](https://github.com/reearth/reearth-web)
-- Backend: [reearth/reearth-backend](https://github.com/reearth/reearth-backend)
+[![stars](https://img.shields.io/github/stars/reearth/reearth)](https://img.shields.io/github/stars/reearth/reearth)
+[![GitHub stars](https://img.shields.io/github/stars/reearth/reearth.svg?style=social&label=Star&maxAge=2592000)](https://github.com/reearth/reearth/stargazers/)
+[![issues](https://img.shields.io/github/issues/reearth/reearth)](https://img.shields.io/github/issues/reearth/reearth)
+[![forks](https://img.shields.io/github/forks/reearth/reearth)](https://img.shields.io/github/forks/reearth/reearth)
+[![](https://img.shields.io/github/license/reearth/reearth)](https://img.shields.io/github/license/reearth/reearth)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/reearth/reearth/graphs/commit-activity)
```

Comment:
Not sure why but "Maintained?" tag is showing twice. Also not sure why here it says "Maintenance" and on the page is says "Maintained", but if possible can you lowercase them? Everything else is lowercase so unless it depends on the tag's source it'll be better!

Reply:
I'll delete one. And lowercased. Thanks.